src/main/resources/otj-pmd-rulesets/ot-pmd.xml

```diff
@@ -73,6 +73,10 @@
         <exclude name="GenericsNaming" />
         <!-- Way too picky @Goofy("star") vs @Goofy(value = "star") is ok -->
         <exclude name="UnnecessaryAnnotationValueElement" />
+        <!-- Too agressive! For example serialNumberUUID is wrong -->
+        <exclude name="FieldNamingConventions" />
+        <!-- misguided attempt to avoid boolean setter getter confusion -->
+        <exclude name="LinguisticNaming" />
```

Comment:
Really, this seems reasonable, what issues did we run into?

Reply:
It hit a lot of fp on david's stuff. As a general rule I'm being conservative on this stuff